addons/sale/views/sale_portal_templates.xml

```diff
@@ -250,17 +250,21 @@
                                     <!-- Display choices only if a pre payment can confirm the order. -->
                                     <div t-if="prepayment_available"
                                          id="o_sale_portal_prepayment_buttons"
-                                         class="d-flex justify-content-center d-grid gap-2 btn-group"
+                                         class="d-flex btn-group"
+                                         role="group"
                                     >
-                                        <button name="o_sale_portal_amount_prepayment_button" class="btn btn-primary rounded">
-                                            Down payment <br/>
+                                        <input type="radio" class="btn-check" name="o_sale_portal_amount_input" id="o_sale_portal_amount_prepayment_input"/>
+                                        <label class="btn btn-light" for="o_sale_portal_amount_prepayment_input">
+                                            Down payment<br/>
                                             <span t-esc="prepayment_amount" class="fw-bold"
                                                   t-options="{'widget': 'monetary', 'display_currency': sale_order.currency_id}"/>
-                                        </button>
-                                        <button name="o_sale_portal_amount_total_button" class="btn btn-primary rounded">
-                                            Full amount <br/>
+                                        </label>
+
+                                        <input type="radio" class="btn-check" name="o_sale_portal_amount_input" id="o_sale_portal_amount_total_input"/>
+                                        <label class="btn btn-light" for="o_sale_portal_amount_total_input">
+                                            Full amount<br/>
                                             <span t-field="sale_order.amount_total"/>
```

Comment:
Why this is not bold like the other one?
**If** it's to de-empathize it then it doesn't really work; it looks more like a glitch rather than like a "different design".